numba_cuda/numba/cuda/tests/doc_examples/test_ffi.py

```diff
@@ -85,6 +85,56 @@ def reduction_caller(result, array):
         actual = r[()]
         np.testing.assert_allclose(expected, actual)
 
+    def test_ex_extra_includes(self):
+        import numpy as np
+        from numba import cuda, config
+        import os
+
+        basedir = os.path.dirname(os.path.abspath(__file__))
+        mul_dir = os.path.join(basedir, "ffi", "include")
+        saxpy_cu = os.path.join(basedir, "ffi", "saxpy.cu")
+
+        testdir = os.path.dirname(basedir)
+        add_dir = os.path.join(testdir, "data", "include")
+
+        old_setting = config.CUDA_RTC_EXTRA_SEARCH_PATHS
```

Comment:
```suggestion
        old_setting = config.CUDA_NVRTC_EXTRA_SEARCH_PATHS
```

Reply:
The `old_setting` should probably be managed by the `override_config` context manager - e.g. 

https://github.com/NVIDIA/numba-cuda/blob/66a7b6fb807418715b9977b26095822a9bad5d8f/numba_cuda/numba/cuda/tests/nrt/test_nrt.py#L91-L92

so that if an error occurs during the test the original setting is restored for the rest of the test suite.

Reply:
I thought of this as well - except I was hoping that the code can demonstrate setting to the `numba.config` code since it's the suggested way to show using it in user code. It's probably ok to wrap the code section with `override config` while still keeping this line.

Reply:
Done in [e1b7c93](https://github.com/NVIDIA/numba-cuda/pull/254/commits/e1b7c93716ee606789a4bf57afe60393a0f100fa), I also verified that afterwards the config is indeed reset to empty string.

Reply:
That makes sense :+1: